docs: clarify purpose and describe ports (#1831)
status-im/status-go: BOOTNODE.md

# Description

> A "Bootnode" is a match-maker for Whisper nodes like a [Mailserver](/MAILSERVER.md). It helps a new node find other nodes it can communicate with. It bootstraps connections between them.

Reviewer: ```suggestion
A "Bootnode" is a match-maker for any type of nodes. It helps a new node find other nodes it can communicate with. It bootstraps connections between them.
```

Author: Fixed.

> A Bootstrap node is a very static thing and should not change it's IP. Their addresses are either hardcoded into the node binary or provded via flags or config files on process start.

Reviewer: ```suggestion
A Bootstrap node is a very static thing and should not change its IP. Their addresses are either hardcoded into the node binary or provided via flags or config files on process start.
```

Author: Fixed.
